joint_trajectory_controller/include/joint_trajectory_controller/joint_trajectory_controller_impl.h

```diff
@@ -790,8 +801,10 @@ setActionFeedback()
   current_active_goal->preallocated_feedback_->desired.positions     = desired_state_.position;
   current_active_goal->preallocated_feedback_->desired.velocities    = desired_state_.velocity;
   current_active_goal->preallocated_feedback_->desired.accelerations = desired_state_.acceleration;
+  current_active_goal->preallocated_feedback_->desired.time_from_start = ros::Duration(desired_state_.time_from_start);
   current_active_goal->preallocated_feedback_->actual.positions      = current_state_.position;
   current_active_goal->preallocated_feedback_->actual.velocities     = current_state_.velocity;
+  current_active_goal->preallocated_feedback_->actual.time_from_start = ros::Duration(current_state_.time_from_start);
   current_active_goal->preallocated_feedback_->error.positions       = state_error_.position;
   current_active_goal->preallocated_feedback_->error.velocities      = state_error_.velocity;
```

Comment:
Let's stick the newly-calculated into the message too

```suggestion
  current_active_goal->preallocated_feedback_->error.velocities      = state_error_.velocity;
  current_active_goal->preallocated_feedback_->error.time_from_start = ros::Duration(state_error_.time_from_start);
```